includes/functions/functions_dates.php

```diff
@@ -262,13 +274,19 @@ function zen_checkdate($date_to_check, $format_string, &$date_array)
  */
 function zen_is_leap_year($year)
 {
-    if ($year % 100 == 0) {
-        if ($year % 400 == 0) return true;
-    } else {
-        if (($year % 4) == 0) return true;
+    if ($year % 4 != 0) {
+        return false;
+    }
+
+    if ($year % 100 != 0) {
+        return false;
+    }
+
+    if ($year % 400 != 0) {
```

Comment:
At this point, it is divisible by 4 and divisible by 100. The rule is though that it is a leap year only if it is also divisible by 400.

So, if it is divisible by 4,  by 100 and not divisible by 400 (non zero remainder when divided by 400), then return false. This check is correct.